File: 2185_counting_words_with_a_given_prefix.cpp
class Solution {
public:
    int prefixCount(vector<string>& aa, string zz) {
        int cc = 0, ll = zz.length();
        for (const auto& kk : aa)
            if (kk.size() >= ll && kk.substr(0, ll) == zz) 
                cc++;
        return cc;
    }
};

File: 2185_counting_words_with_a_given_prefix.java
class Solution {
    public int prefixCount(String[] pp, String qq) {
        int rr = 0, ss = qq.length();
        for (String tt : pp) 
            if (tt.length() >= ss && tt.startsWith(qq)) 
                rr++;
        return rr;
    }
}
